client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -651,6 +651,19 @@ class CancelPurchaseForm extends Component {
 		return translate( 'Cancel product' );
 	}
 
+	getCanceledProduct() {
+		const { purchase, site, translate } = this.props;
+		switch ( purchase.productId ) {
+			// product.Id 5 is Domain Mapping
+			case 5:
+				return ' ' + purchase.productName + translate( ' for ' ) + purchase.meta;
```

Comment:
The leading space returned by this string feels like a code smell. I wouldn't expect a function that returns a string to return a leading space. This indicates that the place where we're rendering this function's result needs a space. This would also make sense if we want to reuse the result of that function somewhere else, where a leading space wouldn't be necessary.

Reply:
Should be resolved by using a placeholders!